certwebapp/settings.py

```diff
@@ -27,7 +27,6 @@
 
 ALLOWED_HOSTS = []
 
-
 # Application definition
 
 INSTALLED_APPS = [
```

Comment:
Add accounts to INSTALLED_APPS:
INSTALLED_APPS = [
    'app',
    'accounts',
....